apps/condo/domains/notification/schema/NotificationUserSetting.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * Generated by `createschema notification.NotificationUserSetting 'user:Relationship:User:CASCADE; messageType:Text; messageTransport:Text; isEnabled:Checkbox'`
+ */
+const dayjs = require('dayjs')
+const get = require('lodash/get')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/notification/access/NotificationUserSetting')
+const { MESSAGE_TRANSPORTS, MESSAGE_TYPES } = require('@condo/domains/notification/constants/constants')
+const {
+    WRONG_MESSAGE_TRANSPORT,
+    WRONG_MESSAGE_TYPE,
+    NO_NEED_TO_ENABLE_NOTIFICATIONS,
+} = require('@condo/domains/notification/constants/errors')
+
+const ERRORS = {
+    WRONG_MESSAGE_TYPE: {
+        code: BAD_USER_INPUT,
+        type: WRONG_MESSAGE_TYPE,
+        message: 'Wrong message type',
+    },
+    WRONG_MESSAGE_TRANSPORT: {
+        code: BAD_USER_INPUT,
+        type: WRONG_MESSAGE_TRANSPORT,
+        message: 'Wrong message transport',
+    },
+    NO_NEED_TO_ENABLE_NOTIFICATIONS: {
+        code: BAD_USER_INPUT,
+        type: NO_NEED_TO_ENABLE_NOTIFICATIONS,
+        message: 'No need to enable notifications. All notifications enabled by default. You may just delete this setting instead.',
+    },
+}
+
+const NotificationUserSetting = new GQLListSchema('NotificationUserSetting', {
+    schemaDoc: 'User controlled notifications settings',
+    fields: {
+
+        user: {
+            schemaDoc: 'The user the settings will apply for',
+            type: 'Relationship',
+            ref: 'User',
+            isRequired: true,
+            knexOptions: { isNotNullable: true }, // Required relationship only!
+            kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
+        },
+
+        messageType: {
+            schemaDoc: 'Affected message type',
+            adminDoc: 'See apps/condo/domains/notification/constants/constants.js:MESSAGE_TYPES for possible values',
```

Comment:
It's better to include all types inside the schemaDoc message by code

Reply:
Done